swiss/src/app/landsraad/landsraad.component.ts

```diff
@@ -77,6 +82,7 @@ export class LandsraadComponent implements OnInit {
   addQuestion(form: NgForm) {
     if (form.valid) {
       let ref = this.db.list("landsraad/questions").push({
+        questionType: "Hlas Landsraadu",
```

Comment:
This should be some constant and not user string

Reply:
Fixed